docs/manual/install.qmd

```diff
@@ -106,12 +106,17 @@ To generate the documentation, you'll first have to:
 
 1. Install [Quarto](https://quarto.org/docs/get-started/)
 2. Activate the Python virtual environment (if any)
-3. Install Python dependencies
+3. Install Python dependencies:
 
     ```bash
     pip install -r requirements-doc.txt
     ```
 
+4. Run `build-api.sh`^[This is currently not very portable, and requires a
```

Comment:
Can you put the full path to the `build-api.sh` file here?

Reply:
done